frontends/ol-search-ui/src/components/LearningResourceCard.tsx

```diff
@@ -0,0 +1,152 @@
+import moment from "moment"
```

Comment:
@abeglova Continuing discussion from Slack... I think `LearningResourceCard` is self-contained enough that it can be moved to `ol-search-ui`.

Re differences between Infinite Corridor and Learn cards, namely:
- different certificate icons
- Learn shows more detail, like subjects and bookmark

I think it would be relatively easy to pass those details in, like:
```tsx
<LearningResourceCard
    resource={resource}
    extraFooterContent={<Bookmark onClick={...} />}            {/*This prop doesn't actually exist, but it could*/}
    certificate={<CustomCertificate />}      {/*Although, maybe better to just use the same cert in Learn and IC*/}
/>
```
Or we could have a `SimpleResourceCard` (learn) and `DetailedResourceCard` (Both in ol-search-ui)... but there's enough here, and it is self contained enough to move to ol-search-ui that I think we should.